host/host_tensor/include/host_gemm.hpp

```diff
@@ -41,3 +41,45 @@ void host_gemm_mk_kn_mn(const Tensor<AType>& a_m_k,
                                c_m_n.mDesc.GetLengths()[0],
                                c_m_n.mDesc.GetLengths()[1])(std::thread::hardware_concurrency());
 }
+
+template <typename AType,
+          typename BType,
+          typename CType,
+          typename AElementwiseOperation,
+          typename BElementwiseOperation,
+          typename CElementwiseOperation>
+void host_gemm_gmk_gkn_gmn(const Tensor<AType>& a_g_m_k,
```

Comment:
Need to remove this, and add a reference operator like this one

https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/823657ed120144943b7db87c07fe3e647128db56/reference_operation/include/reference_gemm.hpp#L19

Reply:
Done